docs/references/creating-a-release.md

```diff
@@ -10,41 +10,86 @@ We must do due diligence in testing our packages before any new major/minor or b
 
-When ready to release to NPM, a new package is created off the release branch and the release branch is then merged into `main` and deleted. A git tag is added to the released git-version.
+  main -.-|Created in step 1| prerelease
+  prerelease -->|Create Branch| release
+  prerelease -.-o|Create Pull Request| main
+```
 
-## Creating a release through GitHub actions (Preferred)
+### Cherry-picking changes
 
-### Getting Ready for release
+While the release branch is active, some changes might be merged into the branch (for bug fixes, features deemed necessary for the release). PRs into the release branch should follow this process when possible:
 
-1. Trigger the "Release branch - Create" GitHub action
-    Enter the branch or tag you are looking to create a release off. This will usually be an alpha tag or the main branch.
+- First land the change as a PR into `main`.
+- Then, cherry-pick the change as a separate PR onto the release branch.
+  - Do not merge changes into the prerelease branch created in Step #1. They may get merged after the release branch is created, and not make it into the release at all.
+  - The release branch is never merged back into `main`, so any changes directly into the release branch will be lost on `main` (and future releases).
 
-    <https://user-images.githubusercontent.com/2684369/130101098-b356b16d-47a7-416b-af84-a27f7e4dc891.mp4>
+This process has the following benefits:
 
-1. The triggered GitHub action will bump the package versions, generate the packlet changelogs and put up a PR up into main.
-1. Double check the package versions are as expected.
-1. The PR will auto create a Storybook deployment that can be used for verifications.
-1. Craft the [`@azure/communication-react` changelog](https://github.com/Azure/communication-ui-library/blob/main/packages/communication-react/CHANGELOG.md).
-    * This should be done as a Pull Request into the release branch.
-    * To create the changelist grab changes from the packlets' changelogs and prune the changelog lines as necessary; ensure the changelog looks good and changelog lines that equate to small PRs for the same feature are combined. For more information see: [Pruning a Changelog](../references/pruning-a-changelog.md).
-    * Example changelog PR: <https://github.com/Azure/communication-ui-library/pull/701>
-1. Ensure the "Before Release" steps of the [release checklist](../references/release-checklist.md) are completed.
+- The release branch never diverges off of `main`. In theory, it is possible to abandon the release branch at any point and create a new one off of `main` without losing work.
+-  All PR reviews happen on `main`, and the cherry-pick PR simply requires a sign-off. This avoids non-trivial merge conflicts when the release branch is eventually merged back into `main`.
 
-### Publishing the package
+## Publishing the package
```

Comment:
Please update line 107: Don't merge release branch back into `main`. Delete it once all release steps are over.